### DISCUSSION TITLE
你好，我下载了项目，本地运行，为什么弄不了动态壁纸啊?

### DISCUSSION THREAD

**Discussion by ChicSparrow** [wangrongding/wallpaper-box#2] [Q&A] [answered]
你好，我下载了项目，本地运行，为什么设置不了动态壁纸啊?
![image](https://user-images.githubusercontent.com/102220342/177253220-090a5885-6bde-4a2e-863a-c6c372f5e1cb.png)

**Comment by wangrongding** [comment, marked as answer]
项目还在开发阶段，把这个文件的32行注释解开，可以体验一下。
https://github.com/wangrongding/desktop-renderer/blob/main/electron/createWallPaper.js#L32
